cluster/environments/azure-multiple-clusters/aks-centralus.tf

```diff
@@ -27,45 +28,53 @@ module "central_vnet" {
   }
 }
 
-# Creates aks cluster
-module "central_aks" {
-  source = "../../azure/aks"
+data "azurerm_client_config" "centralclient" {}
```

Comment:
One only needs to define azurerm_client_config in one location and can reference it in other tf files in the module.

So maybe centralize, `data "azurerm_client_config" "current" {}` in main.tf and just reference...